libraries/engage/core/contexts/AppContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default React.createContext();
```

Comment:
The context is maintained by the AppProvider component, which wraps the content of `theme/pages/index.jsx`. This component initializes the provider. Since it's not intended to be used anywhere else, i think it's ok not to initialize it here.